src/main/java/practice/StreamPractice.java

```diff
@@ -13,8 +20,16 @@ public class StreamPractice {
      * If there is no needed data throw RuntimeException with message
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
+    private static final String NUM_SEPARATOR = ",";
```

Comment:
Maybe we have to define this variable after the class declaration, because we have method description here. To my mind it will be more readable.
```suggestion
public class StreamPractice {
    private static final String NUM_SEPARATOR = ",";
    ...
```